test/liquidity-curve.test.js

```diff
@@ -8,45 +8,59 @@ describe('LiquidityCurve', function () {
     it('saves the points', function () {
       const points = [ [1, 2], [3, 4] ]
       const curve = new LiquidityCurve(points)
-      assert.deepEqual(curve.points, points)
-    })
-  })
-
-  describe('setPoints', function () {
```

Comment:
If `setPoints` is replaced by `setData`, then removing this test is correct, but shouldn't a new one for `setData`  be added in its place?